doc/widgets/scraper.rst

```diff
@@ -0,0 +1,42 @@
+Preprocess
+==========
+
```

Comment:
You have to rename this properly, so the build can link the widget to .rst file.